src/flow/domain/mod.rs

```diff
@@ -593,8 +607,19 @@ impl Domain {
         }
     }
 
-    fn seed_replay(&mut self, tag: Tag, key: &[DataType]) {
-        let (m, source) = match self.replay_paths[&tag] {
+    fn seed_replay(&mut self, tag: Tag, key: &[DataType], transaction_state: Option<ReplayTransactionState>) {
+        if transaction_state.is_none() {
```

Comment:
After discussion, this *should* be `TriggerEndpoint::Start`, and *not* `::Local`.